app/helpers/container_helper/textual_summary.rb

```diff
@@ -4,12 +4,12 @@ module ContainerHelper::TextualSummary
   #
 
   def textual_group_properties
-    items = %w(image name state restart_count backing_ref image_ref)
+    items = %w(name state restart_count backing_ref)
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
   def textual_group_relationships
-    items = %w(ems container_replicator container_group container_project)
+    items = %w(ems container_replicator container_group container_project container_image)
```

Comment:
I think it should not go last here, but rather second - after the provider and before the replicator